spec/asyncapi.md

```diff
@@ -1987,15 +1987,15 @@ schemas:
 
 #### <a name="securitySchemeObject"></a>Security Scheme Object
 
-Defines a security scheme that can be used by the operations. Supported schemes are:
+Defines a security scheme that the operations can use. Supported schemes are:
 
 * User/Password.
-* API key (either as user or as password).
+* API key (either as a user or password).
```

Comment:
I'm not really sure what "either as a user or password" means here... that isn't specific to your change, I wasn't sure what it meant before.